Remotion/Web/Development.WebTesting/HostingStrategies/DockerHosting/DockerCommandLineClient.cs

```diff
@@ -86,20 +89,29 @@ public void Pull (string imageName)
 
       if (mounts.Any())
       {
-        var mountFlags = string.Join(" ", mounts.Select(kvp => $@"-v ""{kvp.Key}"":""{kvp.Value}"""));
+        var mountFlags = string.Join(" ", mounts.Select(kvp => $@"-v ""{kvp.Key.Trim('\\')}"":""{kvp.Value.Trim('\\')}"""));
```

Comment:
```suggestion
        var mountFlags = string.Join(" ", mounts.Select(kvp => $@"-v ""{kvp.Key.Trim('\\')}"":""{kvp.Value.Trim('\\')}"""));
```

The key should only be single-word, no backslashes, correct? It's true that this is defensive coding. The question is if we are trying to make the code robust against formatting issues (e.g. trailing backslash in a path) or protect it against injection of exploits. I would suggest only protecting against formatting, thus the key would be taken at face value, no quotes, not handling of special characters.